### ISSUE TITLE
Riding whale animation isn't synced properly

### ISSUE THREAD

**Issue by bud3699** [ismail0234/Subnautica-Below-Zero-Multiplayer#264]
**Which version of the launcher are you using?**
-  (v6.6.6 etc.)

**using v1.0.5**

**Which platform are you playing the game on?**
- (Steam/Epic Games/Xbox Game Pass)
**steam**

**Explain your problem**
- (Explain in as much detail as possible.)
**when riding the whale near the lilly pads, the user gets stuck in the riding animation for everyone**

**How can we reproduce this Bug?**
- (Describe the steps required to generate the bug.)
**Ride the whale**

**Log Files**
- (Attach the log files here.)
**Long gone, deleted them, my bad**

**Screenshots**
- (Fill in if available.)
![image](https://github.com/user-attachments/assets/bef668e5-c6b9-449f-b013-5017eee06c3c)

**Video Records**
- (Fill in if available.)#
n/a

**Comment by ismail0234** [comment]
Describe the problem reproduction step in more detail.

**Comment by bud3699** [comment]
All i did was ride the whale in the lilypads. For the person who rides it, the animations return to normal, for the rest of the people, they're stuck with the animation

I did also have a weird bug with the spy pengling, once i used it it just would not let me use any of my tools until i rejoined (Like couldnt equip them)

**Comment by ismail0234** [comment]
> All i did was ride the whale in the lilypads. For the person who rides it, the animations return to normal, for the rest of the people, they're stuck with the animation
> 
> I did also have a weird bug with the spy pengling, once i used it it just would not let me use any of my tools until i rejoined (Like couldnt equip them)

I can't reproduce the problem. There's no problem. Can you try to reproduce the problem?

**Comment by bud3699** [comment]
> > All i did was ride the whale in the lilypads. For the person who rides it, the animations return to normal, for the rest of the people, they're stuck with the animation
> > I did also have a weird bug with the spy pengling, once i used it it just would not let me use any of my tools until i rejoined (Like couldnt equip them)
> 
> I can't reproduce the problem. There's no problem. Can you try to reproduce the problem?

Yup, I'll give it a go later, I'm about to head out into the city, It happened multiple times on the same save, I didnt try on a different one, I'll send updated info when i get back

**Comment by bud3699** [comment]
I cant seem to re-create the issue on a new save, but its persistent on one save. So i guess close it or delete the issue and i'll let you know if i can consistently re-create it. I did find another bug which i'll record and open up a new issue

**Comment by ismail0234** [comment]
It's a strange problem. It can't be reproduced all the time. But sometimes this problem occurs. I'll look into it a little bit.

**Comment by ismail0234** [comment]
Fixed. (Beta Version)
